cypress/support/api/utils/discounts/salesUtils.js

```diff
@@ -15,42 +16,61 @@ export function createSaleInChannel({
   type,
   value,
   channelId,
-  discountValue = value
+  discountValue = value,
 }) {
   let sale;
   return createSale({
     name,
     type,
-    value
+    value,
   })
     .then(saleResp => {
       sale = saleResp;
       addChannelToSale(sale.id, channelId, discountValue);
     })
     .then(() => sale);
 }
+export function shouldVariantBeOnSale(
```

Comment:
```suggestion
export function getVariantWithSale({
	variantId,
	channelSlug,
	shouldBeSales,
	retries
})
```

Reply:
I thought about that - but I use it also to confirm that variant is no longer on Sale.

Reply:
confirmVariantSaleStatus - maybe this is more descriptive?

Reply:
what about `getVariantWithStatus({status: 'onSale'})`?